doc/tools/west/repo-tool.rst

```diff
@@ -327,5 +327,47 @@ These commands perform miscellaneous functions.
   that fetches upstream data is run (this behavior can be
   suppressed for the duration of a single command by passing ``--no-update``).
 
+.. _west-mr-build:
+
+Build system integration
+************************
+
+Some of the projects managed by west require to be integrated with the Zephyr
+build system, which is based around Kconfig and CMake (see
+:ref:`application` for more information about Zephyr's build system).
+
+The way that the build system discovers and includes west-managed projects when
+building an application is based around the ``west list`` command. The main
+CMake script in zephyr (the :file:`CMakeLists.txt` located at the zephyr
+repository root folder) invokes ``west list`` in order to obtain a list of
+projects managed by west.
+
+The CMake code in :file:`CMakeLists.txt` retrieves the following
+information for each project using ``west list``:
+- name: The name of the project as specified in the manifest file
+- path: The path of the project within the west installation
+
+Once it has collected the list, the CMake script performs the following
+operations on each project:
+
+- If a matching :file:`CMakeLists.txt` is located, it will process it directly
```

Comment:
`zephyr/CMakeLists.txt` and `zephyr/Kconfig`, right? Or did you change this decision after I left the meeting?

Reply:
Also, this seems a bit redundant with the below paragraph that says the same thing.

Reply:
I will rework this because it is clearly confusing